build/Versions.props

```diff
@@ -21,8 +21,6 @@
     <MicrosoftDotNetIBCMergeVersion>4.7.1-alpha-00001</MicrosoftDotNetIBCMergeVersion>
     <MicrosoftNetCompilersVersion>3.3.0-beta1-19327-04</MicrosoftNetCompilersVersion>
     <MicrosoftDiaSymReaderPdb2PdbVersion>1.1.0-beta1-62624-01</MicrosoftDiaSymReaderPdb2PdbVersion>
-    <RoslynAnalyzersPackagesVersion>2.9.3</RoslynAnalyzersPackagesVersion>
```

Comment:
These properties are supposed to be used when we use them in targets files and project files, I've just inlined the versions similar to other package references.